pkg/sql/codegen/intermediate_representation.go

```diff
@@ -70,10 +70,6 @@ type SingleSQLIR interface {
 }
 
 // TrainIR is the intermediate representation for code generation of a training job.
-//
-// Please be aware that the TrainIR intentionally excludes the model table name in the
-// INTO clause. The sql package will save the output files of a generated Python program.
-// For prediction and analysis jobs, the sql will restore an identical working directly.
```

Comment:
SQLFlow generates a workflow program. The workflow program will save the model. So the `TrainIR` should contain `Into`.